nimare/meta/kernel.py

```diff
@@ -159,7 +161,8 @@ def transform(self, dataset, masker=None, return_type="image"):
 
         if not isinstance(transformed_maps[0], (list, tuple)):
             if return_type == "array":
-                return transformed_maps[0][:, mask_data]
+                masked_ma_arr = transformed_maps[0][:, mask_data]
+                return masked_ma_arr
```

Comment:
```suggestion
                return transformed_maps[0][:, mask_data]
```

This looks like it can be reverted to what it was before